gbfs-validator/__test__/fixtures/conditionnal_no_vehicle_type_id.js

```diff
@@ -1,5 +1,7 @@
 const fastify = require('fastify')
 
+const last_updated_fresh = Math.floor(Date.now() / 1000) - 30
```

Comment:
As a general rule of thumb, testing based on an actual clock could lead to unexpected results and false negatives / positives. Using a mocked clock is preferable.